javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,62 @@
+let boughtCandyPrices = [];
+
+function addCandy(candyType , weight) {
+  if(candyType === 'Sweet')
+    boughtCandyPrices.push(weight*0.5);
+  
+  else if(candyType === 'Chocolate') 
+    boughtCandyPrices.push(weight*0.7);
+
+  else if(candyType === 'Toffee') 
+    boughtCandyPrices.push(weight*1.1);
+  
+  else if(candyType === 'Chewing-gum') 
+    boughtCandyPrices.push(weight*0.03);
```

Comment:
It might be a better idea to simply return the candy value from this funcion, rather then pushing it to array from here. Ideally this code would be organized in more functions `checkCandyValue`, `buyCandy`, `canBuyMoreCandy` - then each funcion would do only one thing wich is usually how we want the code to work. It makes it easier to maintain and read.